setup.cfg

```diff
@@ -20,7 +20,7 @@ default_section = THIRDPARTY
 testpaths = tests
 filterwarnings =
     error
-    ignore:the imp module:DeprecationWarning:
+    ignore::DeprecationWarning
```

Comment:
Without this test are failing on python3.8:
```
.../python3.8/site-packages/aiohttp/helpers.py:107: in <module>
    def noop(*args, **kwargs):  # type: ignore
.../python3.8/asyncio/coroutines.py:111: in coroutine
    warnings.warn('"@coroutine" decorator is deprecated since Python 3.8, use "async def" instead',
E   DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
=============================================================================================== short test summary info ================================================================================================
ERROR tests/e2e/test_main.py - DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=================================================================================================== 1 error in 0.23s ===================================================================================================
make: *** [test_e2e] Error 2
```

Reply:
That's fine.
I have to find time for publishing Python 3.8 compatible aiohttp release